Workbooks/WorkbooksMetadata.json

```diff
@@ -2159,7 +2159,7 @@
     "dataTypesDependencies": ["SentinelHealth"],
     "dataConnectorsDependencies": [],
     "previewImagesFileNames": [ "AutomationHealthBlack.png", "AutomationHealthWhite.png" ], 
-    "version": "1.0.0",
+    "version": "2.0.0",
```

Comment:
@BenjiSec, hope the version updated is correct. Can you please check if it should be 1.0.1?, Thanks.

Reply:
@v-sabiraj is ti possible to get this approved today as we need to push for deployment into Sentinel GUI?